### PR TITLE
Remove Facebook and Twitter share buttons in Drop flow

### PR DESCRIPTION
**Pull request by Boerworz**
`AllihoopaCore` no longer supports sharing to Facebook and Twitter, so this pull requests removes the buttons from the UI.

This pull request also updates the `AllihoopaCore` dependency to version `4.0.0`.

### CODE REVIEW

File: Allihoopa/Drop/DropViewController.m [hunk 1]
```diff
@@ -202,7 +202,7 @@ - (void)segueToCancelledViewController {
     [self cancelDrop];
 }
 
-- (void)segueToErrorViewController {
+- (void)segueToErrorViewController:(NSError *)error {
```
Comment (JagCesar) [inline Allihoopa/Drop/DropViewController.m]: Is this really related to this PR? Any why is this argument necessary?

Comment (Boerworz) [reply Allihoopa/Drop/DropViewController.m]: The method signature of this method was changed to take the error argument in version 4.0.0 of `AllihoopaCore`, which we're using as of this pull request.

**Comment by Boerworz** [comment]
Good catch, I'll update the README now!

File: Allihoopa/Drop/DropViewController.m (hunk 1 above)
Comment (JagCesar) [reply Allihoopa/Drop/DropViewController.m]: Awesome. 👍
